src/CattyTests/Bricks/PreviousLookBrickTests.swift

```diff
@@ -44,9 +43,9 @@ final class PreviousLookBrickTests: AbstractBrickTest {
         var look2: Look!
         do {
             look = Look(name: "test", andPath: "test.png")
-            try imageData?.write(to: URL(fileURLWithPath: object.projectPath() + "images/test.png"))
+            try imageData?.write(to: URL(fileURLWithPath: object.projectPath() + "Scene 1/images/test.png"))
             look2 = Look(name: "test2", andPath: "test2.png")
-            try imageData?.write(to: URL(fileURLWithPath: object.projectPath() + "images/test2.png"))
+            try imageData?.write(to: URL(fileURLWithPath: object.projectPath() + "Scene 1/images/test2.png"))
```

Comment:
Please do not use the hardcoded name of the scene here. I would suggest to use `project.scene.imagesPath` instead (or similar).